<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select a Feature to Split Data</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>mole.png</normaloff>mole.png</iconset>
  </property>
  <widget class="QListWidget" name="featureListWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="featureListTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Click Feature to Split</string>
   </property>
  </widget>
  <widget class="QLabel" name="uniqueCountLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Unique Count: N/A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="splitByFeatureButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Split by Feature</string>
   </property>
  </widget>
  <widget class="QListWidget" name="uniqueListWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="countWarningLabel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Count should be less than 10</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
